webui/react/package.json

```diff
@@ -24,18 +24,21 @@
     "dayjs": "^1.10.4",
     "eventemitter3": "^4.0.7",
     "fp-ts": "^2.8.4",
+    "fuse.js": "^6.4.6",
     "history": "^4.10.1",
     "humanize-duration": "^3.24.0",
     "io-ts": "^2.2.11",
     "js-sha512": "^0.8.0",
     "js-yaml": "^3.14.0",
+    "omnibar": "github:hamidzr/omnibar",
```

Comment:
capturing what we talked about in terms of packaging the omnibar:
- start by bringing in the entire omnibar project into `determined` codebase
- making the necessary customizations to the omnibar project
- keep the omnibar package mostly isolated (including the tweaks) so we are able to break it out into a separate repo of its own once it has stablized

Reply:
I think both TS and eslint are gonna be very unhappy.
we can ignore eslint for the whole directory but ts would be more challenging. I'll look into this more.